examples/prediction/HousingServe.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import joblib
+import fairing
+import numpy as np
+
+MODEL_FILE = 'trained_ames_model.dat'
+
+fairing.config.set_builder('docker',
+registry='gcr.io/mrick-gcp',
+base_image="seldonio/seldon-core-s2i-python3:0.4")
+
+fairing.config.set_deployer('serving', serving_class="HousingServe")
+fairing.config.run()
```

Comment:
(Not blocking) Would the location of `fairing.config.run()` change depending on the preprocessor that's used? It's a bit odd to see it before the class definition.

Reply:
Nope. I put it here because since it running everything remotely, its faster to put it closer to the top (so heavy libraries arent loaded before it gets ran). But in reality it can be any where

Reply:
It was confusing for me too. I think just for clarity we can place it at the end of the file.